### PR TITLE
Fix info

### PR DESCRIPTION
**Pull request by anushks**
corrected info

### CODE REVIEW

File: src/components/Examples.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const builtApps = [
   {
     title: 'Roomie McRoomFace',
-    description: 'A room booking system for the UCL Engineering Hub. If you\'re an undergraduate student part of the engineering faculty, you can log in and start booking rooms.',
+    description: 'A room booking system for the UCL Engineering Hub. If you\'re an undergraduate student part of the engineering faculty, you can log in and start booking rooms. Fun fact: this was actually built before the API existed!',
```
Comment (ChristopherHammond13) [inline src/components/Examples.jsx]: Doesn't this mean that we're lying then?

Comment (ChristopherHammond13) [reply src/components/Examples.jsx]: @anushks

Comment (anushks) [reply src/components/Examples.jsx]: minor tbh

Comment (anushks) [reply src/components/Examples.jsx]: we could just remove it

Comment (ChristopherHammond13) [reply src/components/Examples.jsx]: up to you tbh

**Pull request merged**